### PR TITLE
Work order functional requirements

### PR DESCRIPTION
**Pull request by arkinmodi**
Close https://github.com/arkinmodi/project-sayyara/issues/71

### CODE REVIEW

**Review by TimChoy — APPROVED** [review]

File: docs/SRS/SRS.tex [hunk 1]
```diff
@@ -338,6 +334,156 @@ \subsubsection{Authentication}
...
+		            \begin{enumerate}
+			            \item[] N/A
+		            \end{enumerate}
+		      \item Viewpoint: Auto Repair Shop Owner
+		            \begin{enumerate}
+			            \item The system shall deleted the associated work order
+		            \end{enumerate}
+		      \item Viewpoint: Auto Repair Shop Employee
+		            \begin{enumerate}
+			            \item The system shall deleted the associated work order
```
Comment (joyxiao99) [inline docs/SRS/SRS.tex:379]: Delete

File: docs/SRS/SRS.tex [hunk 2]
```diff
@@ -338,6 +334,156 @@ \subsubsection{Authentication}
...
+
+	\item An appointment has been cancelled
+	      \begin{enumerate}[VP\arabic*.]
+		      \item Viewpoint: Vehicle Owner
+		            \begin{enumerate}
+			            \item[] N/A
+		            \end{enumerate}
+		      \item Viewpoint: Auto Repair Shop Owner
+		            \begin{enumerate}
+			            \item The system shall deleted the associated work order
```
Comment (joyxiao99) [inline docs/SRS/SRS.tex:375]: Delete

File: docs/SRS/SRS.tex [hunk 3]
```diff
@@ -338,6 +334,156 @@ \subsubsection{Authentication}
...
+	\item The user wants to view the details of a work order
+	      \begin{enumerate}[VP\arabic*.]
+		      \item Viewpoint: Vehicle Owner
+		            \begin{enumerate}
+			            \item[] N/A
+		            \end{enumerate}
+		      \item Viewpoint: Auto Repair Shop Owner
+		            \begin{enumerate}
+			            \item The system shall list the shop details, services to be performed with their individual bill rates
+			                  and expected number of hour for completion, parts required and their cost, odometer reading before
```
Comment (joyxiao99) [inline docs/SRS/SRS.tex:472]: “Hours”

File: docs/SRS/SRS.tex [hunk 4]
```diff
@@ -338,6 +334,156 @@ \subsubsection{Authentication}
...
+			                  and after service, customer details, assigned employee, car details, any applied discounts, final
+			                  balance for the customer, warranty information, digital vehicle inspection, and any extra notes
+		            \end{enumerate}
+		      \item Viewpoint: Auto Repair Shop Employee
+		            \begin{enumerate}
+			            \item The system shall list the shop details, services to be performed with their individual bill rates
+			                  and expected number of hour for completion, parts required and their cost, odometer reading before
```
Comment (joyxiao99) [inline docs/SRS/SRS.tex:479]: “Hours”

File: docs/SRS/SRS.tex [hunk 5]
```diff
@@ -338,6 +334,156 @@ \subsubsection{Authentication}
...
+	\item An appointment has been scheduled
+	      \begin{enumerate}[VP\arabic*.]
+		      \item Viewpoint: Vehicle Owner
+		            \begin{enumerate}
+			            \item[] N/A
+		            \end{enumerate}
+		      \item Viewpoint: Auto Repair Shop Owner
+		            \begin{enumerate}
+			            \item The system shall create a work order
+			            \item The system shall populate the customer data and vehicle data from quote
```
Comment (joyxiao99) [inline docs/SRS/SRS.tex:348]: For this section do you want to say “the quote” and “the appointment”

Comment (TimChoy) [reply docs/SRS/SRS.tex:348]: I think quote makes more sense here, the appointment is the actual event, but the quote has the details and pricing

Comment (arkinmodi) [reply docs/SRS/SRS.tex:348]: So basically merging these two lines?
```latex
\item The system shall populate the customer data and vehicle data from quote
\item The system shall populate the customer data and vehicle data from appointment if quote not
```

Comment (joyxiao99) [reply docs/SRS/SRS.tex:348]: adding the word "the"

**Pull request merged**